eta-meeting-organizer-backend/src/main/java/hu/flowacademy/meetingorganizer/rest/MeetingRoomResource.java

```diff
@@ -0,0 +1,52 @@
+package hu.flowacademy.meetingorganizer.rest;
+
+import hu.flowacademy.meetingorganizer.persistance.model.MeetingRoom;
+import hu.flowacademy.meetingorganizer.service.MeetingRoomService;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api")
+@AllArgsConstructor
+public class MeetingRoomResource {
+
+    private MeetingRoomService meetingRoomService;
+
+    @GetMapping("/meetingrooms")
```

Comment:
Hogyha minden handler metódus path-a tartalmazza a `/meetingrooms`-t, miért nem adjuk meg a `@RequestMapping`-be (class szinten)?

> Less code is a feature.